pom.xml

```diff
@@ -14,7 +14,9 @@
     <url>https://github.com/jenkinsci/yaml-axis-plugin</url>
     <properties>
         <changelist>999999-SNAPSHOT</changelist>
-        <jenkins.version>2.414.3</jenkins.version>
+        <!-- https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/ -->
+        <jenkins.baseline>2.479</jenkins.baseline>
+        <jenkins.version>${jenkins.baseline}.1</jenkins.version>
```

Comment:
@strangelookingnerd https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/#currently-recommended-versions says the following

> At the moment, the Jenkins releases 2.462.3 and 2.479.3 make good core dependencies. 

Is the reason you chose 2.479.1 instead of 2.479.3 here because [2.479.3](https://www.jenkins.io/changelog-stable/#v2.479.3) is too new?

Reply:
No particular reason actually. Bumped to `.3`

Reply:
No particular reason, bumped to `.3`.